### DISCUSSION TITLE
Icon/Brand

### DISCUSSION THREAD

**Discussion by samholmes** [ghostty-org/ghostty#2887] [Q&A] [answered]
I'm creating this issue to track the issue about icon/brand.

I've been contributing icons on the Discord since before the invite to this repo. Since then, I've been working with the community there about what contribution I can make when it comes to the official Ghostty icon and brand. So, I'm officially sharing the progress on that and opening up the discussion the repo maintainers to voice their input on this development with the hope that I can make an official contribution to the icon design and branding of Ghostty.

So without further ado...

# Matilda

I've made a ghost whom I named Matilda as a mascot for Ghostty. She's a cutie. He name is inspired by the tilde character (~) because it's the ghostiest character on the keyboard.

<img width=360 src=https://github.com/user-attachments/assets/3ef11f21-6d63-480a-bf4c-1ea09c958d25 />

# Icon

## Basic design

<img alt="basic" width="360" src=https://github.com/user-attachments/assets/8f5686c2-5049-4c1e-996a-80407b3f6ec6 />

### Solid background
This is inspired gasoline coloring and theme. It's a spooky look.

<img alt="ghastly" width="360"  src=https://github.com/user-attachments/assets/7a9173c3-437b-4c18-9848-c9e833aca5c4 />

### Clear background
In addition to this theme, I've designed a transparent version which could be a unique idea for an icon.

<img alt="ghastly clear" width="360"  src=https://github.com/user-attachments/assets/5bb20a63-25e4-4318-bfdc-718867a4da33 />

### Fancy

<img width="360" alt="Ghostty (fancy)" src="https://github.com/user-attachments/assets/3c61f3f2-eeda-4970-9849-1278a2ba2de2">

## Modern

<img width="360" alt="Ghostty Modern" src="https://github.com/user-attachments/assets/2899598b-a04a-4bfd-9482-607905d4065d">

# Proposal

I propose the use of colors from this icon design (purple, blue, green, black). These colors could be apart of a color scheme used beyond the icon (website, t-shirt branding, etc).

[The Challenger Deep](https://challenger-deep-theme.github.io/) theme is closest to this color scheme and can be forked and tweaked into a custom default color scheme for the terminal as well.

This is a message to the maintainers of this repo and the community to vote on whether this design concept should be developed further. I'm opening up the discussion to get feedback to guide the development of this icon and brand. I look forward to hearing from the community.

**Comment by mitchellh** [comment, marked as answer]
Hello! I converted to a discussion. An issue is for directly actionable work (as noted in CONTRIBUTING.md). Discussions are for open ended ongoing well... discussions. 

Thanks for your passion around this! I'm sorry to say that I've been working with a few designers (some friends, some hired) to shore up our branding. This isn't a final version but you can see and in-progress version here: https://x.com/mitchellh/status/1857131140035023049 The icon is still a work in progress and I'm not ready to share yet, but it is more or less the same icon we have but with a lot more detail fleshed out and matching the new ghost. 

I appreciate the passion you put into these icons, but I plan on keeping your work to date as a community piece of work and not adopting it officially.
